capstone/src/main/webapp/form-loader.js

```diff
@@ -0,0 +1,113 @@
+var CLIENT_ID = '1002281613740-gsmtp3tteqgtoformikuglhu8fr388et.apps.googleusercontent.com';
+var API_KEY = 'AIzaSyBS13DWNfA-RpO4AQrd4tdUE15-chVDbJc';
+var DISCOVERY_DOCS = ["https://script.googleapis.com/$discovery/rest?version=v1"];
+var SCOPES = 'https://www.googleapis.com/auth/forms';
```

Comment:
I will probably do another push to delete the keys or work with Kyle to create a credentials file